store/zookeeper/zookeeper.go

```diff
@@ -13,7 +13,9 @@ const (
 	// SOH control character
 	SOH = "\x01"
 
-	defaultTimeout = 10 * time.Second
+	defaultTimeout = 10 * time.Second	
+
+	syncRetryLimit = 5
```

Comment:
@dongluochen No specific reason, except in my test environment 3  retries would always be needed to get the correct value, and I rounded this up to 5. 
Note that the sync currently happens on SOH or empty string getting returned, and not on all reads. Also this is to work around older versions of libkv which have this bug. The versions that have this patch will do create+write atomically and should not return SOH or empty so the sync should never be needed.